scripts/gen_top_tags_per_country.pl

```diff
@@ -208,6 +214,16 @@
 		}
 	}
 
+	# Add environmental impact from impact estimator if we have them
+	if ((defined $product_ref->{ecoscore_extended_data}) and (defined $product_ref->{ecoscore_extended_data}{impact})
+		and (defined $product_ref->{ecoscore_extended_data}{impact}{likeliest_impacts})
+		# Need to add a filter to keep only impacts computed with high confidence
```

Comment:
the comment is not clear to me. Is this a TODO ?

Reply:
yes, I'll add TODO